label_studio/io_storages/azure_blob/models.py

```diff
@@ -209,28 +209,16 @@ def iterkeys(self):
                 continue
             yield file.name
 
-    def get_data(self, key) -> Union[dict, list[dict]]:
+    def get_data(self, key) -> tuple[list[dict], list[int | None], list[int | None]]:
```

Comment:
Once we're at a 3-tuple return type, I think it'd be better to create a simple object with a constructor and return that, or use a solution like NamedTuple or TypedDict. Python errors from unpacking tuples into too few variables in assignment statements are quite common

Reply:
also, for the `list[int | None]` types, I think it'd be better to say `list[int] | Literal[[None]]`, since there is no other list containing `None`s that we could want returned afaik.

Reply:
Not a bad idea. I'm thinking `(task_data: list[dict] (or pa.Table), storagelink_params: list[StorageLinkParam])`

Reply:
though tbf, this function will only ever have one caller

Reply:
done